<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1567</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PD_Soft_V2</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::LeftToRight</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>179</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>50</height>
     </size>
    </property>
    <property name="title">
     <string>时基</string>
    </property>
    <widget class="QSpinBox" name="sb_timebase">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5000</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_Timebase">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>145</y>
      <width>179</width>
      <height>110</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>110</height>
     </size>
    </property>
    <property name="title">
     <string>参数</string>
    </property>
    <widget class="QComboBox" name="cb_Couping">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>26</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_Voltage">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>53</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_Channel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>30</y>
       <width>53</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Coupling</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>54</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Voltage</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>82</y>
       <width>53</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>261</y>
      <width>179</width>
      <height>80</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>80</height>
     </size>
    </property>
    <property name="title">
     <string>采集</string>
    </property>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>99</x>
       <y>21</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>停止</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>21</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>51</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>清除缓存</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_15">
     <property name="geometry">
      <rect>
       <x>99</x>
       <y>51</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>后台采集</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>347</y>
      <width>179</width>
      <height>80</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>80</height>
     </size>
    </property>
    <property name="title">
     <string>模拟</string>
    </property>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>单帧渲染</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>开始模拟数据</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>94</x>
       <y>22</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>停止模拟数据</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>433</y>
      <width>179</width>
      <height>120</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>120</height>
     </size>
    </property>
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>加载CVS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>92</x>
       <y>60</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>导出CVS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>加载文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>加载缓存文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>92</x>
       <y>90</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>导出缓存文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>92</x>
       <y>30</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>保存文件</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>559</y>
      <width>179</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Debug Log</string>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>1084</x>
      <y>10</y>
      <width>481</width>
      <height>801</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>谐波</string>
     </attribute>
     <widget class="QTableView" name="tableView2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>471</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>410</y>
        <width>471</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>387</y>
        <width>53</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>缓存</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>352</y>
        <width>471</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>导出谐波计算结果数据</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="harmonicParam">
      <property name="geometry">
       <rect>
        <x>73</x>
        <y>10</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>2000000000</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>13</y>
        <width>53</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>基波频率</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>13</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Hz   (0为软件检测基础频率，如不准确可自定义)</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_CacheCount">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>387</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>384</y>
        <width>401</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>67</y>
      <width>179</width>
      <height>72</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>72</height>
     </size>
    </property>
    <property name="title">
     <string>信号</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>53</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>测试</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>46</y>
       <width>53</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>信号幅值</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_fvalues">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_vvalues">
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>46</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>15</y>
      <width>881</width>
      <height>791</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Time</string>
     </attribute>
     <widget class="PDChart" name="chartView" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>871</width>
        <height>761</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Frequency</string>
     </attribute>
     <widget class="PDChart" name="chartView_2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>871</width>
        <height>761</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>HarmonicRatio</string>
     </attribute>
     <widget class="PDChart" name="chartView_3" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>871</width>
        <height>761</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>HarmonicMagnitude</string>
     </attribute>
     <widget class="PDChart" name="chartView_4" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>871</width>
        <height>761</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>772</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1567</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionopen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionopen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PDChart</class>
   <extends>QWidget</extends>
   <header location="global">pdchart.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
